client/lib/localforage/index.js

```diff
@@ -16,11 +21,15 @@ const config = {
 	]
 };
 
+let _ready = false;
 // Promise that resolves when our localforage configuration has been applied
-const localForagePromise = new Promise( ( resolve ) => {
-	localforage.config( config );
-	resolve( localforage );
-} );
+const localForagePromise = localforage.defineDriver( localforageBypass )
+	.then( () => {
+		localforage.config( config );
```

Comment:
Hmm, that's weird, are you seeing a `Cannot bypass localforage after initialization` error in the console?